tests/test_bot_player_trade.js

```
@@ -0,0 +1,135 @@
+const assert = require('assert');
+
+require('../src/Global');
+
+const Database = invoke('Database');
+const Item = invoke('GameServer/Item/Item');
+const BotTradeService = invoke('GameServer/Bot/BotTradeService');
```

ESLint

[error] 'BotTradeService' is assigned a value but never used.

(no-unused-vars)